taipy/gui/utils/_evaluator.py

```diff
@@ -198,6 +201,7 @@ def __get_holder_hash(holder: t.Type[_TaipyBase], expr_hash: str) -> str:
         return f"{holder.get_hash()}_{_get_client_var_name(expr_hash)}"
 
     def __evaluate_holder(self, gui: Gui, holder: t.Type[_TaipyBase], expr: str) -> t.Optional[_TaipyBase]:
+        expr_hash = ""
```

Comment:
Why do we need the empty declaration here

Reply:
it is used in the except block